helm/chaos-mesh/templates/controller-manager-deployment.yaml

```diff
@@ -53,8 +53,10 @@ spec:
             mountPath: /etc/webhook/certs
             readOnly: true
         ports:
-          - name: webhook-api
+          - name: webhook
             containerPort: 9443 # Customize containerPort
+          - name: http
+            containerPort: 8080
```

Comment:
Can we switch to an infrequently used port? eg: 20180

Reply:
May I ask any difference between 8080 and 20180? Since we are not using hostport here and any access to controller-manager is via Service. So I think any port is okay. I am not sure if I miss something.

Reply:
I think `8080` is used for common service and this port may be used by chaos-dashboard or the other services. At here, we just need to export our metric.

Reply:
Metric is also an HTTP endpoint. And the default value is 8080 as well https://github.com/pingcap/chaos-mesh/blob/09afa9d7074b88bdb143e6f8bcbdd9e7b69c4b3c/cmd/controller-manager/main.go#L67. Do you think we need to change it?

Reply:
I think we can change it.

Reply:
OK. I will update them in this PR

Reply:
Updated. Please take a look